e2e-tests/run-pr.csv

```diff
@@ -4,6 +4,7 @@ config
 config-router
 demand-backup
 gr-demand-backup
+gr-demand-backup-haproxy
```

Comment:
please add this to all `run-*.csv` files

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/435/commits/a999fad6b15851af20df3ad440927a90a57027b5